src/realm/sync/network/network_ssl.hpp

```diff
@@ -188,6 +188,8 @@ class Context {
 /// until its completion handler starts executing.
 class Stream {
 public:
+    struct MockSSLError;
```

Comment:
Shouldn't this and the friend below be declared under REALM_HAVE_SECURE_TRANSPORT? Everything else is